### ISSUE TITLE
[Feature Request] [Valorant] Telemetry

### ISSUE THREAD

**Issue by deankostomaj** [RiotGames/developer-relations#270]
**Feature description**  
A telemetry file in a JSON like format that would include all events that happened during the game including players' positions.   

The most important events in my opinion:  
* players' position  
* players' shots  
* players' abilities used  
* plant of the Spike  
* players' health  
* players' kills  
* players' hits  

Those are just the most important events if I forgot any just them down under comments.

**Benefits**  
This kind of feature has unlimited potential for developers to come up with solutions to enrich Valorant community and give players/analysts a lot of data to analyze. This leads to more competitive and more enjoyable matches and also a bigger audience.

**Comment by CeH9** [comment]
Yeah, would be nice to hhave something like [this](https://developer.valvesoftware.com/wiki/Counter-Strike:_Global_Offensive_Game_State_Integration)

**Comment by deankostomaj** [comment]
good point @CeH9, would be an awesome feature, but I was thinking more something like [this](https://documentation.pubg.com/en/telemetry.html)

**Comment by zmagajna** [comment]
I think it will be an awesome feature, see what is possible with PUBG API which provides telemetry to us  (Twire) [https://twire.gg/]

**Comment by hexIV** [comment]
This would be one of the best thing to have.

Game Servers push events to a Firebase environment in JSON and who has access can listen to these events.  Once the match is finished all the data can be removed after X minutes and will be available in a separate API endpoint with the breakdown of the data in general in a `/match/match-id` endpoint (can be less detailed)
